templates/statistics/index.html.twig

```diff
@@ -0,0 +1,18 @@
+{# statistics/index.html.twig #}
+
+{% extends 'base.html.twig' %}
+
+{% block page %}
+
+  <h3>Statistiques</h3>
+  <ul>
+    <li><a href='{{ asset("statistics/time") }}'>Feuille de temps</a></li>
+    <li><a href='{{ asset("statistics/agent") }}'>Par agent</a></li>
+    <li><a href='{{ asset("statistics/service") }}'>Par service</a></li>
+    <li><a href='{{ asset("statistics/status") }}'>Par statut</a></li>
+    <li><a href='{{ asset("statistics/position") }}'>Par poste</a></li>
+    <li><a href='{{ asset("statistics/positionsummary") }}'>Par poste (Synthèse)</a></li>
+    <li><a href='{{ asset("statistics/supportposition") }}'>Postes de renfort</a></li>
```

Comment:
You should not set href with routes that don't exist yet. When i said "use asset", i mean with old url. This way:
<li><a href='{{ asset("index.php?page=statistiques/temps.php") }}'>Feuille de temps</a></li>

Reply:
asset("index.php?page=statistiques/temps.php")